src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -4317,7 +4317,7 @@ public class Test
         }
 
         [Fact]
-        public void CS0306ERR_BadTypeArgument01()
+        public void CS0306ERR_BadTypeArgument01_CSharp12()
```

Comment:
This test (and some others below) are named `CSharp12` but don't use a locked langver. I think those tests should use an explicit langver (ie. `RegularNext`)

Reply:
Done.